ratelimiter/Dockerfile

```diff
@@ -34,6 +34,7 @@ ENV GRPC_PORT=8091
 ENV USE_STATSD=false
 ENV LOG_LEVEL=INFO
 ENV LOCAL_CACHE_SIZE_IN_BYTES=1024000
+ENV STOP_CACHE_KEY_INCREMENT_WHEN_OVERLIMIT=true
```

Comment:
shall we add a new cucumber test case for this issue, if possible?

Reply:
Added the test case from a35af18f7d2e4b9111f3f7abf7745186e6290e4d